pkg/review/commit_review_status.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package review contains code to get review status information
+// for a GitHub commit.
+
+package review
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/shurcooL/githubv4"
+)
+
+// PullRequest represents a pull request in GitHub and contains the
+// GitHub assigned ID, the pull request number in the repository,
+// and the review decision for the pull request
+type PullRequest struct {
+	DatabaseID     githubv4.Int
+	Number         githubv4.Int
+	ReviewDecision githubv4.String
+}
+
+// Retrieves all associated pull requests for a commit from GitHub based on
+// the given GitHub organization, repository, and commit sha. If the commit
+// has no associated pull requests than an empty slice is returned.
+func GetPullRequests(client *githubv4.Client, githubOrg, repository, commitSha string) ([]PullRequest, error) {
```

Comment:
Godoc requires you to start the comment with func name
```suggestion
// GetPullRequests retrieves all associated pull requests for a commit from GitHub based on
// the given GitHub organization, repository, and commit sha. If the commit
// has no associated pull requests than an empty slice is returned.
func GetPullRequests(client *githubv4.Client, githubOrg, repository, commitSha string) ([]PullRequest, error) {
```

Reply:
fixed.